src/main/java/org/openjdk/jextract/impl/Writer.java

```diff
@@ -26,64 +26,28 @@
 
 package org.openjdk.jextract.impl;
 
-import javax.tools.JavaFileObject;
+import org.openjdk.jextract.JavaSourceFile;
```

Comment:
Do we really need a class for this? Seems more like an helper method to the main jextract tool?

Reply:
You mean remove `Writer`? Or `JavaSourceFile`?

Reply:
`Writer`